analyze_foldamers/ensembles/cluster.py

```diff
@@ -196,13 +216,23 @@ def get_cluster_medoid_positions_KMedoids(
             n_clusters, cluster_rmsd, cluster_sizes, plotfile
             )        
     
-    return (medoid_positions, cluster_sizes, cluster_rmsd, silhouette_avg)
+    if return_original_indices:
```

Comment:
I agree, I didn't want to mess up the current tests with the new outputs, but multiple number variable outputs gets confusing. Maybe we could do an output where it returns none if it's not requested via flag?

Reply:
Yeah - I say instead of returning None we can just have it always return the original indices and labels, those are generally pretty useful to have. We'll need to update the tests too.

Reply:
Okay! That sounds good!

Reply:
I can do the same for the silhouette average output we have. I think that was also contributing to the multiple number of return statements